app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -15,11 +15,14 @@ const DeleteAlert = ({
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      logger.log(selectedNoteIds);
```

Comment:
Why do we need to log this here? Please don't push your testing logic into the PR. These unnecessary logging statements ought to be removed as part of your self-review.